.github/workflows/flatpak-build.yaml

```diff
@@ -0,0 +1,193 @@
+name: Flatpak Build
+on:
+  workflow_call:
+    inputs:
+      arch:
+        default: x86_64
+        type: string
+      build-bundle:
+        default: true
+        type: boolean
+      bundle:
+        default: obs-studio-flatpak.flatpak
+        type: string
+jobs:
+  build-until-90:
+    name: Build Modules Until 90-* 🧾
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+    outputs:
+      cacheKey: ${{ steps.setup.outputs.cacheKey }}
+      cacheHit: ${{ steps.setup.outputs.cacheHit }}
+      qemuArch: ${{ steps.setup.outputs.qemuArch }}
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:kde-6.5
+      options: --privileged
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          fetch-depth: 0
+          set-safe-directory: ${{ env.GITHUB_WORKSPACE }}
+
+      - name: Set Up Environment 🔧
+        id: setup
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          : Set Up Environment 🔧
+          if [[ "${RUNNER_DEBUG}" ]]; then set -x; fi
+
+          git config --global --add safe.directory "${GITHUB_WORKSPACE}"
+
+          echo '::group::Install GitHub CLI tool'
+          dnf install -y -q gh
+          gh extension install actions/gh-actions-cache
+          echo '::endgroup::'
+
+          if [[ "${{ inputs.arch }}" != "x86_64" ]]; then
```

Comment:
Change some things but I prefer future-proofing with x86_64 as host than hard-coding to only aarch64 and x86_46.

Reply:
Just for the sake of completeness:

flatpak-github-actions seeks to avoid dependency on docker (in docker) and the setup-qemu action.

I proposed a solution upstream https://github.com/flatpak/flatpak-github-actions/pull/173, I'm just waiting for feedback from maintainers.